cmd/harmony/config.go

```diff
@@ -151,6 +153,18 @@ type prometheusConfig struct {
 	Gateway    string
 }
 
+type syncConfig struct {
+	LegacyServer   bool
```

Comment:
can we add some comments on what these fields mean?

Reply:
Sure. Added.